### PR TITLE
[feat]: Add Apache.maven

### LINKED ISSUE #65391: [Package Request]: Apache Maven
### Package Requested

- [X] I would like someone else to build the manifest.
- [ ] I would like help so I can submit the manifest.
- [X] I have performed a search and couldn't find this package.
- [ ] I have checked that the installer for this package is not a standalone or portable executable or a .zip file
- [ ] I think there is a new version available and I have provided the URL.

### Please provide the following information

Publisher: The Apache Software Foundation
Publisher Url: https://www.apache.org/
Package Name: Apache Maven
Description: Apache Maven is a tool that can now be used for building and managing any Java-based project.
Package Version: 3.8.6
Installer URL: https://dlcdn.apache.org/maven/maven-3/3.8.6/binaries/apache-maven-3.8.6-bin.zip

Current Package Identifier (for new versions): 

>Apache Maven 3.3+ requires JDK 1.7 or above to execute

### PR DESCRIPTION
**Pull request by soroshsabz**
ITNOA

This PR add maven to winget

Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue? fixes #65391 

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/189469)

### CODE REVIEW

**Comment by soroshsabz** [comment]
@vedantmgoyal9 I need a little help

when I try to test this new manifest, I see below log in install stage

```
> winget install --manifest 3.9.9 --ignore-local-archive-malware-scan

Found Apache Maven [Apache.Maven] Version 3.9.9
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Successfully verified installer hash
Archive scan detected malware. Proceeding due to --ignore-local-archive-malware-scan
Extracting archive...
Successfully extracted archive
Starting package install...
Command line alias added: "mvn"
Successfully installed
```

after install successfully, I can see mvn symbolic link in below

```
    Directory: C:\Users\sooro\AppData\Local\Microsoft\WinGet\Links

Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
la---           11/7/2024  7:47 PM              0 mvn.exe -> C:\Users\sooro\AppData\Local\Microsoft\WinGet\Packages\Apa
                                                  che.Maven__DefaultSource\apache-maven-3.9.9\bin\mvn.cmd
```

**My first question is, why generate mvn.exe?** I want to see mvn.cmd

another problem is if I run original file like below

```
C:\Users\sooro\AppData\Local\Microsoft\WinGet\Packages\Apache.Maven__DefaultSource\apache-maven-3.9.9\bin\mvn.cmd
```

everything is ok

but if I run empty mvn in shell, I see one error and then see another window that show original maven logs

image

How to resolve this?

**Comment by soroshsabz** [comment]
@denelon Do you can review my PR? and see my question?

thanks

File: manifests/a/Apache/Maven/3/3.9.9/Apache.Maven.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+InstallerLocale: en-US
+InstallerType: zip
+Commands:
+- mvn
+ReleaseDate: 2024-08-17
+Installers:
+- Architecture: neutral
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: apache-maven-3.9.9/bin/mvn.cmd
```
Comment (Trenly) [inline manifests/a/Apache/Maven/3/3.9.9/Apache.Maven.installer.yaml:15]: CMD files are not allowed by policy

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by Trenly** [comment]
[Policy] Scripted Application

File: manifests/a/Apache/Maven/3/3.9.9/Apache.Maven.installer.yaml (hunk 1 above)
Comment (soroshsabz) [reply manifests/a/Apache/Maven/3/3.9.9/Apache.Maven.installer.yaml]: @trenly maven publish officially mvn.cmd for windows, How to can I do?

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Comment by soroshsabz** [comment]
@denelon why close this PR?

**Comment by denelon** [comment]
.cmd isn't currently allowed due to our policies for the community repository.

I'm talking with some of our security folks about the possibility of supporting them in addition to .exe.

* https://github.com/microsoft/winget-cli/issues/5085

**Pull request closed without merging**